src/track/frame.h

```diff
@@ -0,0 +1,102 @@
+#pragma once
+
+class QDebug;
+
+namespace mixxx {
+class Frame {
+  public:
+    Frame()
+            : m_dFrame(0) {
+    }
+
+    explicit Frame(double dFrame)
+            : m_dFrame(dFrame) {
+    }
+
+    ~Frame() = default;
+
+    void setValue(double dFrame) {
+        m_dFrame = dFrame;
+    }
+
+    double getValue() const {
+        return m_dFrame;
+    }
+
+    Frame& operator+=(const Frame& arg) {
+        m_dFrame += arg.getValue();
+        return *this;
+    }
+
+    Frame& operator-=(const Frame& arg) {
+        m_dFrame -= arg.getValue();
+        return *this;
+    }
+
+    Frame& operator*=(const double arg) {
+        m_dFrame *= arg;
+        return *this;
+    }
+
+    Frame& operator/=(const double arg) {
+        m_dFrame /= arg;
+        return *this;
+    }
+
+    Frame operator++(int) {
```

Comment:
Please omit these operators, operators +=/-= are sufficient and more versatile. The value is not supposed to be an integer.